lab/src/Microsoft.ServiceModel.Syndication/src/SyndicationFeed.cs

```diff
@@ -302,14 +446,14 @@ public Rss20FeedFormatter GetRss20Formatter(bool serializeExtensionsAsAtom)
             return new Rss20FeedFormatter(this, serializeExtensionsAsAtom);
         }
 
-        public void SaveAsAtom10(XmlWriter writer)
+        public async Task SaveAsAtom10Async(XmlWriter writer, CancellationToken ct)
         {
-            this.GetAtom10Formatter().WriteTo(writer);
+            await this.GetAtom10Formatter().WriteToAsync(writer, ct);
         }
```

Comment:
Don't use `this.` when redundant. This is part of the coding standards for open source .Net